apps/meteor/server/startup/migrations/v305.ts

```diff
@@ -0,0 +1,25 @@
+import { Settings } from '@rocket.chat/models';
+import { isValidCron } from 'cron-validator';
+
+import { settings } from '../../../app/settings/server';
```

Comment:
We should not use the cached setting in migrations, let's use the model instead.